frontend/src/app/main/store.cljs

```diff
@@ -89,7 +89,9 @@
      (letfn [(show-shape [shape-id level objects]
                (let [shape (get objects shape-id)]
                  (println (str/pad (str (str/repeat "  " level)
-                                        (:name shape))
+                                        (:name shape)
+                                        (when (seq (filter #(not= :position-group %)
```

Comment:
the seq + filter can be replaced with `some`

Reply:
I've just realized that i removed :position-group. So it can be simplified.